### ISSUE TITLE
Closing TODOs does not log as per logdone

### ISSUE THREAD

**Issue by meedstrom** [200ok-ch/organice#980]
**Describe the bug**
I wondered if Organice supports logging when TODO states change to DONE.  I ran across the log-into-drawer setting (PR #271), and its documentation seems to imply that organice supports it (though the main readme does not mention it).  Yet it never happens in my files.

I've tried re-installing the Organice app, and a single file repo with minimal content:

```
# -*- org-log-done: t; -*-
#+STARTUP: logdone
* DONE Apply on CSN.se
* DONE clone organice repo to laptop
* TODO clean up here in prep for going to alno
* TODO Summarize history on Logseq
```

**Expected behavior**
I expect that when I change TODO to DONE, a line will appear containing e.g. "CLOSED: [2024-04-17 Wed 15:03]".

**Smartphone**
 - Device: iPhone SE 2020
 - OS: iOS 17.3.1
 - Browser: PWA (from Safari)

**Comment by schoettl** [comment]
Hi Martin, I think logging of TODO state changes is only implemented for recurring tasks. The `#+STARTUP:` line is probably parsed by organice but not used. So we would consider this issue as a missing feature.

Would you like to work on this? It could be a nice first issue.

**Comment by vHugoObject** [comment]
Can I take this?

**Comment by munen** [comment]
@vHugoObject Thank you for the offer🙏🏻 Good luck, have fun👍

**Comment by meedstrom** [comment]
Yeah, I didn’t get around to it. If you make half-progress and stall, feel
free to share your findings and we could finish together 😉
